packages/ui-tests/cypress/fixtures/flows/errorHandlerCR.yaml

```diff
@@ -0,0 +1,11 @@
+- errorHandler:
```

Comment:
@tplevko, I added this `errorHandlerCR` file as a starting point for the e2e, does this help?

Reply:
Sure @lordrip, looks like a good starting point. Thanks!